monkey/infection_monkey/monkey.py

```diff
@@ -113,6 +116,7 @@ def __init__(self, args):
 
         self._singleton = SystemSingleton()
         self._opts = self._get_arguments(args)
+        self._server_strings = [str(s) for s in self._opts.servers]
```

Comment:
why can't we use a list of socket addresses instead?

Reply:
This was a stopgap used when not all methods used `SocketAddress`. However, it appears that it is not longer required